doc/en/components/grids/_shared/column-hiding.md

```diff
@@ -84,13 +85,408 @@ Let's start by creating our `{ComponentName}` and binding it to our data. We wil
     <IgrColumn field="Phone" dataType="string" sortable="true"></IgrColumn>
 </IgrGrid>
 ```
+<!-- ComponentEnd: Grid -->
+
+<!-- ComponentStart: HierarchicalGrid -->
```

Comment:
this is a bit of an overkill. Lets just add a snipet showing exactly what we explained, instead of the whole grid.
we binding to data -> data=*
we adding sorting -> sorting=true
All we need is to look something like this:
<hgrid data=*
            sorting=*
            filtering=*>
</hgrid>
The whole sample should be in example area where ppl can check the code further